build/config/compiler/BUILD.gn

```diff
@@ -283,9 +283,12 @@ config("compiler") {
 
     cflags += common_mac_flags
 
-    # Without this, the constructors and destructors of a C++ object inside
-    # an Objective C struct won't be called, which is very bad.
-    cflags_objcc = [ "-fobjc-call-cxx-cdtors" ]
+    cflags_objcc = [
+      "-std=c++11",
```

Comment:
I printed all the cflags_cc and I get `["-fno-threadsafe-statics", "-fvisibility-inlines-hidden", "-Wno-undefined-bool-conversion", "-Wno-tautological-undefined-compare", "-std=c++11"]`. How did you find -fno-exceptions in cflags_cc?